modules/ipi-install-configuring-raid-for-worker-node.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id="configuring-raid-for-worker-node_{context}"]
+= Configuring RAID for worker node
+
+The following procedure configures RAID (Redundant Array of Independent Disks) for the worker node during the installation process.
+
+[NOTE]
+====
+. Only servers with BMC (Baseboard Management Controller) type `irmc` are supported. Other types of servers are currently not supported.
+. If you want to configure hardware RAID for the server, make sure the server must have a RAID controller.
+====
+
+.Procedure
+. Create manifests.
+. Modify the BMH (Bare Metal Host) file corresponding to the worker:
++
+[source,terminal]
+----
+$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-3.yaml
+----
++
+[NOTE]
+====
+Because servers with BMC type `irmc` do not support software RAID, the following RAID configuration uses hardware RAID as an example.
+====
++
+.. If you added specific RAID configuration to the spec, this will cause the worker node to delete the original RAID configuration
+in the `preparing` phase and perform a specified configuration on the RAID. For example:
++
+[source,yaml]
+----
+spec:
+  raid:
+    hardwareRAIDVolumes:
+    - level: "0"
+      name: "sda"
+      numberOfPhysicalDisks: 1
+      rotational: true
+      sizeGibibytes: 0
+----
+<1> `level` is a required field, and the others are optional fields.
++
+.. If you added an empty RAID configuration to the spec, this empty configuration causes the worker node to delete the original RAID configuration during the `preparing` phase, but will not perform a new configuration. For example:
```

Comment:
Change to present tense: "...the `preparing` phase, but does not perform a new configuration."